tools/reviewer/local_server/web_login_js/firebase.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Cool! Maybe we can further simplify by putting all the js inside the html file?
That way you can actually open the file in the browser and it will work, you don't need a server.
Or is that frowned upon? :-)

Reply:
Good idea :)

Reply:
Looks like we can't work with firebase, when all the code is launched from a single file.
Error:
```
This operation is not supported in the environment…chrome-extension and web storage must be enabled.
```

Reply:
But the file work, if it's launched from server.
So I can put everything in one file, but there's no much sense in it.

Reply:
Ok, let's still keep it in one file. I think it will be easier to host it that way.